airgun/views/template.py

```diff
@@ -0,0 +1,42 @@
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Text
+from widgetastic.widget import TextInput
+from widgetastic.widget import View
+
+from airgun.views.common import BaseLoggedInView
+from airgun.widgets import ActionsDropdown
+from airgun.widgets import RadioGroup
+
+
+class SyncTemplatesView(BaseLoggedInView):
+    title = Text("//h2[contains(., 'Import or Export Templates')]")
+    import_export = RadioGroup("//div[@class='col-md-6']")
```

Comment:
I do not have any issues with this locator, good way to add like e.g. https://github.com/SatelliteQE/airgun/blob/ab272c64a23d485c0932ea5b5bafd624042028cf/airgun/views/host.py#L353
`action_type = RadioGroup(locator="//div[label[contains(., 'Action type')]]") `